packages/roosterjs-editor-core/lib/corePlugins/DOMEventPlugin.ts

```diff
@@ -236,7 +236,7 @@ export default class DOMEventPlugin implements PluginWithState<DOMEventPluginSta
             }
         });
         this.editor?.triggerPluginEvent(PluginEventType.ContextMenu, {
-            rawEvent: event,
+            rawEvent: event as PointerEvent,
```

Comment:
can we only change content model code, but leave the old editor code not changed?

Reply:
I reverted the change in old editor code.